Sources/MongoSwift/MongoClient.swift

```diff
@@ -202,7 +197,9 @@ public class MongoClient {
     public let writeConcern: WriteConcern?
 
     /**
-     * Create a new client connection to a MongoDB server.
+     * Create a new client connection to a MongoDB server. For options that included in both the connection string URI
+     * and the ClientOptions struct, the final value is set in descending order of priority: the value specified in
+     * ClientOptions (if non-nil), the value specified in the URI, or the server’s default value if both are unset.
```

Comment:
nit: maybe just say "default" instead of "server's default" since some options .e.g `ReadPreference` are actually a driver default 
same applies to other usages